aws_lambda_powertools/event_handler/api_gateway.py

```diff
@@ -128,16 +132,29 @@ def __init__(
         allow_credentials: bool
             A boolean value that sets the value of `Access-Control-Allow-Credentials`
         """
-        self.allow_origin = allow_origin
```

Comment:
I saw that you needed to change the test and I'm Just wondering if it doesn't make sense to keep this field for compatibility issues. I'm not sure if anyone has used this in their local tests and we can break something or it's better to ask for forgiveness than permission.

For me we can remove that but that's just a question.

Reply:
As we've agreed on a call, we're going to:

- keep the removal of the undocumented property, and do a quick rollback if anyone is impacted 
- prefix the new variables with "_" so we can start signaling private properties